contracts/lib/IPresale.sol

```diff
@@ -10,4 +10,5 @@ interface IPresale {
     function refund(address _contributor, uint256 _vestedPurchaseId) external;
     function contributionToTokens(uint256 _value) external view returns (uint256);
     function contributionToken() external view returns (ERC20);
+    function isClosed() external view returns (bool);
```

Comment:
Just wondering; is this something that we can change now with the fundraising presale and etc. already deployed?

Reply:
> Just wondering; is this something that we can change now with the fundraising presale and etc. already deployed?

I think so, as both the Fundraising presale contract and the Court presale one have `isClosed` implementation.

> (I also don't see where it's used in the `UniswapWrapper.sol` contract, although it does seem to be used in #9).

Yes, that's another point for the split version :wink: I'll remove it from here and leave it in #7